ci/docker/Dockerfile.build.jetson

```diff
@@ -42,7 +42,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     automake \
     libtool \
     crossbuild-essential-arm64 \
- && rm -rf /var/lib/apt/lists/*
+ && rm -rf /var/lib/apt/lists/* \
+ && alias python=python3
```

Comment:
This won't help as the alias will be lost at the end of this RUN command. If you don't plan to fix the onnx update in this PR, you can manually install python2 https://packages.ubuntu.com/focal/python2

Reply:
Yep. Let me revoke the previous changes related to python and then add the python2.